src/wired/config.rs

```diff
@@ -90,7 +92,7 @@ impl Default for Config {
         Self {
             verbose: 0,
             lnp2p_addr: InetSocketAddr::default(),
-            monitor_addr: MONITOR_ADDR_DEFAULT.parse().expect("Constant default value parse fail"),
+            monitor_addr: monitor_addr_default(),
```

Comment:
Not sure why we need this as a function, not a string constant parsed here in place. I'd like to have all defaults in the same place, single file, as constants.

Reply:
Ah, my strong habit of statically preventing panics creeped here. :) (You may notice that the function returns the constructed type without ever panicking.)

I can change it if you believe that parsing constant string and unwrapping it is cleaner.

Reply:
Well, defaults may change, and if they are spread across the code it can be tricky. Even if it will panic, it will do it at launch time, so will be explicitly seen by the dev, and here it may be better to stick to "Erlang" approach of failing fast.